README.md

```diff
@@ -49,7 +49,7 @@ Run `docker system prune -a`
 
 Build Leonardo and run all unit tests.
 ```
-export SBT_OPTS="-Xmx2G -Xms1G -Dmysql.host=localhost -Dmysql.port=3311 -Duser.timezone=UTC"
+export JAVA_OPTS="-Dheadless=false -Duser.timezone=UTC"
```

Comment:
why is this change?

Reply:
I don't think `SBT_OPTS` works anymore..probably due to sbt version upgrade